test/test_extended_models.py

```diff
@@ -155,21 +155,26 @@ def test_schema_meta_validation(model_fn):
         "recipe",
         "unquantized",
         "_docs",
+        "_flops",
+        "_weight_size",
     }
     # mandatory fields for each computer vision task
     classification_fields = {"categories", ("_metrics", "ImageNet-1K", "acc@1"), ("_metrics", "ImageNet-1K", "acc@5")}
+    rich_metadata = {"_flops", "_weight_size"}
```

Comment:
I know we discussed offline to put them here. I think the `_weight_size` should be part of `"all"` as it's very easy to estimate. Could you add it on all quantized models?

Reply:
Will add it.